src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -36,7 +37,7 @@ export type SendConfirmationStateProps = {
   pending: boolean,
   keyboardIsVisible: boolean,
   balanceInCrypto: string,
-  balanceInFiat: string,
+  balanceInFiat: number,
   parentDisplayDenomination: EdgeDenomination,
```

Comment:
Balance in fiat will be a number since exchange rates are a number which need to be multiplied by.